packit_service/worker/handlers/distgit.py

```diff
@@ -461,6 +464,19 @@ def pre_check(self) -> bool:
                         f"configuration: {self.job_config.allowed_committers}."
                     )
                     return False
+        elif self.data.event_type in (PullRequestCommentPagureEvent.__name__,):
```

Comment:
I can see that we are doing similar thing here (I am implementing #1541). Can you please move this `elif` block to the method into `JobHandler` superclass so I can later use it in `CreateBodhiUpdateHandler`? Just moving this block of code to separated method in the superclass is enough... I will update this method later to fit the needs of both classes.

(Or if you want, replacing the `koji builds` part in the stings with some variable should be enough)

Reply:
When I have fixed it, as Laura wrote maybe I am doing the wrong check now, I will move it in a separate method.

Reply:
I moved the check in a separate method, but in the same class. I am not really sure you should reuse this check. I think that not every packager (as is for the `koji-build`) should be able to do a `bodhi-update` but I may be wrong.